### DISCUSSION TITLE
Disabled Entry widget not in TTK Creator

### DISCUSSION THREAD

**Discussion by inyoka** [israel-dryer/ttkbootstrap#446] [Q&A]
I've been building an app which is a glorified calculator.  It uses standard ttk.Entry in a disabled state to display results.  This works great in all 'light' themes, however dark themes (Darkly, Solar, Cyborg and Vapor for example) tend to have not really applied appropriate colours to disabled entry widgets.  Part of the problem appears to be that TTK Creator doesn't show a disabled Entry element.  

Is there an easy way for me to correct these display issues?  Below shows examples of the readability issues with a "Flatly" example for contrast.
Flatly (no problem) ...
![image](https://user-images.githubusercontent.com/6580673/229053653-66db6d88-fad2-447c-a47e-690f55c4c83e.png)
Vapour ...
![image](https://user-images.githubusercontent.com/6580673/229053264-edfbafff-9a32-4fd1-8e92-a53c9d6cf0de.png)
Cyborg ...
![image](https://user-images.githubusercontent.com/6580673/229053425-18474adb-98d2-4c51-acac-9c31674a935a.png)
Solar ...
![image](https://user-images.githubusercontent.com/6580673/229053498-96647ff1-6a55-4120-99ab-71b5e07b63b3.png)
Darkly ...
![image](https://user-images.githubusercontent.com/6580673/229053578-3d51916d-9351-4de3-b3fd-0e7e0c98e355.png)

Many thanks for any help.

**Comment by israel-dryer** [comment]
@inyoka, have you tried using 'readonly' instead of 'disabled'?
